IdentityCore/src/webview/systemWebview/ios/MSIDSystemWebviewController.m

```diff
@@ -56,7 +57,8 @@ - (instancetype)initWithStartURL:(NSURL *)startURL
         return nil;
     }
     
-    if (!callbackURLScheme)
+    NSURL *redirectURL = [NSURL URLWithString:redirectURI];
+    if (!redirectURL)
```

Comment:
nit: can you also check for scheme presence here as technically scheme is nullable on NSURL